tests/engine/processes/workchains/test_restart.py

```diff
@@ -67,8 +73,10 @@ def test_get_process_handlers_by_priority(generate_work_chain, inputs, prioritie
     # Verify the actual handlers on the class haven't been modified
     assert getattr(SomeWorkChain, 'handler_a').priority == 200
     assert getattr(SomeWorkChain, 'handler_b').priority == 100
+    assert getattr(SomeWorkChain, 'handler_c').priority == 0
     assert getattr(SomeWorkChain, 'handler_a').enabled
     assert getattr(SomeWorkChain, 'handler_b').enabled
+    assert not getattr(SomeWorkChain, 'handler_c').enabled
```

Comment:
If I understand correctly, this `enabled` is the handler's enabled not the override's enabled?

Reply:
Exactly, I added this check to make sure that the logic that handles overrides does not accidentally override the default value of the handler, which would change behavior in the next iteration of the workchain.